app/entities/Product.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Entities;
+
+class Product {
```

Comment:
keine setter 👍

Reply:
"if it has any setters, it is not an Entity" :D 
Bei Entitäten geht es um verhalten